reanalysis/reanalysis_global.toml

```diff
@@ -1,6 +1,7 @@
 [buckets]
 web_suffix = "web"
 tmp_suffix = "tmp"
+analysis_suffix = "analysis"
```

Comment:
Do we really need to make this configurable? Particularly if we can resolve https://github.com/populationgenomics/automated-interpretation-pipeline/issues/122#issuecomment-1276894986 more generally?

Reply:
Possibly not, but it permits a user to leave out that index completely if we want `suffix=None` to be passed to `output_path`, depending on infrastructure setup.

I definitely think it can be improved upon, e.g. overriding the bucket names using @illusional 's format string suggestion

Reply:
Sorry, only catching up with this discussion. 

I like Michael's suggested toml section. But noting that there is some existing code for cloud paths in cpg_utils, which we can extend upon, or reuse the existing terminology. Happy to look into that more next week